corehq/apps/users/bulkupload.py

```diff
@@ -650,8 +654,6 @@ def _set_progress(progress):
             ) % (_error_message, e.errors))
             ret['errors'].append(_error_message)
 
-    if Domain.get_by_name(domain).supports_multiple_locations_per_user:
-        create_or_update_locations(domain, location_specs, log=ret)
```

Comment:
Yikes, I didn't even know we did this.  That function should be able to be removed now.